### DISCUSSION TITLE
Icons all seemed to be broken?

### DISCUSSION THREAD

**Discussion by isaaclem90** [nvim-tree/nvim-tree.lua#1874] [Q&A]
### Description

I've just freshly installed both nvim and nvim-tree but the icons seems to be broken? Kindly refer to the image below 
![image](https://user-images.githubusercontent.com/70257036/210080113-81e344d1-d4e2-4751-8960-7ebe4c4825b5.png)

Below is the configuration I have on iterms2 and that the font & icons are able to load perfectly normal in the terminal itself. 

Not sure if anyone can guide me through on setting it up?
![image](https://user-images.githubusercontent.com/70257036/210080187-3295c31d-a390-42cd-8470-8e2218ee2662.png)

### Neovim version

```text
NVIM v0.8.2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@Monterey
```

### Operating system and version

macOS Monterey 12.6.2

### nvim-tree version

Latest

### Minimal config

```lua
local setup, nvimtree = pcall(require, "nvim-tree")
if not setup then
  return
end

-- recommended settings from nvim-tree documentation
vim.g.loaded = 1
vim.g.loaded_netrwPlugin = 1

nvimtree.setup()
```

### Steps to reproduce

1. Fresh installed nvim-tree, toggle on and noticed all icons are broken

**Comment by alex-courtis** [comment]
What was the problem/solution?

It would be useful for future users who encounter a similar issue

**Comment by Alan-MQ** [comment]
have u solved this one ?

**Reply by gegoune** [reply]
As you can see on attached screenshot no nerd font is in use, which is required to display icons. Please search other (including closed) issues and discussions.
